compiler-embeddable/src/test/kotlin/it/unibo/collektive/test/TestCompileString.kt

```diff
@@ -34,7 +34,7 @@ class TestCompileString : FreeSpec({
                 disassembled shouldNot beNull()
                 disassembled shouldNot beBlank()
                 val alignedOnCalls = disassembled.lines().filter {
-                    "// InterfaceMethod it/unibo/collektive/aggregate/api/Aggregate.alignedOn:" in it
+                    "// InterfaceMethod it/unibo/collektive/aggregate/api/Aggregate.alignRaw:" in it
```

Comment:
```suggestion
                    "// InterfaceMethod it/unibo/collektive/aggregate/api/Aggregate.align" in it
```

This should work also if we decide to switch back